CHANGELOG.md

```diff
@@ -13,9 +13,10 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 
 ### Added
 
-- [#3133](https://github.com/thanos-io/thanos/pull/3133) Query: Allow passing a `storeMatch[]` to Labels APIs. Also time range metadata based store filtering is supported on Labels APIs.
+- [#3133](https://github.com/thanos-io/thanos/pull/3133) Querier: Allow passing a `storeMatch[]` to Labels APIs. Also time range metadata based store filtering is supported on Labels APIs.
 - [#3154](https://github.com/thanos-io/thanos/pull/3154) Query Frontend: Add metric `thanos_memcached_getmulti_gate_queries_max`.
```

Comment:
Wait, why this is related to `Query Frontend` :laughing:  Would you mind updating it in this pr?

Reply:
It doesn't change anything in `Query Frontend`. Just renamed a changelog entry.

Reply:
Why the changelog title is `Query Frontend`, I am confused... It is `pkg/gate`, so should be `Querier`?

Reply:
Now, I get it. I was focusing on the changes that I've made. I can fix it as well.